### DISCUSSION TITLE
wrong specifications on margin-block-end

### DISCUSSION THREAD

**Discussion by mwmcode** [mdn/mdn-community#549] [Content] [answered]
https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block-end

refers to scroll snap

![Screen Shot 2022-08-31 at 11 19 13 AM](https://user-images.githubusercontent.com/13101565/187716458-0cc661b7-108e-45ce-8c37-6cca4e1b60e4.png)

apologies if this is the wrong place/category. I tried to fork/edit but these variables are fed externally.

**Comment by teoli2003** [comment, marked as answer]
Thank you very much. I opened a PR to get this fixed at mdn/browser-compat-data#17626 (It will take 1 week before it is live on the site, as the data is stored in a package that is updated once a week).

As this is an issue, not a discussion, can you open an issue next time (it is quicker for you, and easier for us to track problems).

On each page, there is a box at the bottom with the direct link to open an issue:
![Capture d’écran 2022-08-31 à 18 08 31](https://user-images.githubusercontent.com/1466293/187726433-881bfa02-6283-411b-bb7a-a9f5a383143c.png)


Thank you very much.

**Reply by mwmcode** [reply]
Glad I could be of help. You're right, I don't know why I didn't open an issue for this. Will do next time. 
Thank you for the quick action
